builder/src/firmware.rs

```diff
@@ -15,48 +15,118 @@ pub fn rom_from_env() -> &'static FwId<'static> {
     }
 }
 
+#[cfg(feature = "hw-latest")]
+pub const ROM: FwId = FwId {
+    crate_name: "caliptra-rom",
+    bin_name: "caliptra-rom",
+    features: &["hw-latest"],
+};
+
+#[cfg(not(feature = "hw-latest"))]
 pub const ROM: FwId = FwId {
     crate_name: "caliptra-rom",
     bin_name: "caliptra-rom",
     features: &[],
 };
 
+#[cfg(feature = "hw-latest")]
+pub const ROM_WITH_UART: FwId = FwId {
+    crate_name: "caliptra-rom",
+    bin_name: "caliptra-rom",
+    features: &["emu", "hw-latest"],
+};
+
+#[cfg(not(feature = "hw-latest"))]
 pub const ROM_WITH_UART: FwId = FwId {
     crate_name: "caliptra-rom",
     bin_name: "caliptra-rom",
     features: &["emu"],
 };
 
+#[cfg(feature = "hw-latest")]
+pub const ROM_FAKE_WITH_UART: FwId = FwId {
+    crate_name: "caliptra-rom",
+    bin_name: "caliptra-rom",
+    features: &["emu", "fake-rom", "hw-latest"],
+};
+
+#[cfg(not(feature = "hw-latest"))]
 pub const ROM_FAKE_WITH_UART: FwId = FwId {
     crate_name: "caliptra-rom",
     bin_name: "caliptra-rom",
     features: &["emu", "fake-rom"],
 };
 
+#[cfg(feature = "hw-latest")]
+pub const FMC_WITH_UART: FwId = FwId {
+    crate_name: "caliptra-fmc",
+    bin_name: "caliptra-fmc",
+    features: &["emu", "hw-latest"],
+};
+
+#[cfg(not(feature = "hw-latest"))]
 pub const FMC_WITH_UART: FwId = FwId {
     crate_name: "caliptra-fmc",
     bin_name: "caliptra-fmc",
     features: &["emu"],
 };
 
+#[cfg(feature = "hw-latest")]
+pub const FMC_FAKE_WITH_UART: FwId = FwId {
+    crate_name: "caliptra-fmc",
+    bin_name: "caliptra-fmc",
+    features: &["emu", "fake-fmc", "hw-latest"],
```

Comment:
Seeing as we're doing this to pretty much every target I wonder if it we be more maintainable to revert the changes to this file and instead decide to add the hw-latest feature to every target in build_firmware_elfs_uncached().

Reply:
Ok.  I will make the change.

Reply:
Done.